packages/web-react/src/components/FileUploader/README.md

```diff
@@ -293,6 +310,7 @@ The rest of the properties are created from the default `<div>` element. [More a
 | `maxFileSize`      | `number`                             | 1000000 | ✕        | The maximum size of the uploaded file                                                                                                                           |
 | `maxUploadedFiles` | `number`                             | 10      | ✕        | Maximum file upload queue size                                                                                                                                  |
 | `isMultiple`       | `boolean`                            | -       | ✕        | When multiple files can be selected at once                                                                                                                     |
+| `isDismissible`    | `boolean`                            | -       | ✕        | When to hide FileUploaderInput after reaching file limit in `maxUploadedFiles`                                                                                  |
```

Comment:
How about something more extensible:

```suggestion
| `queueLimitBehavior`    | `disable`, `hide` | `hide` | ✕        | Behavior of FileUploaderInput after reaching file limit in `maxUploadedFiles`                                                                                  |
```

1. We can add more options in the future.
2. IMO, [_dismiss_](https://slovnik.seznam.cz/preklad/anglicky_cesky/dismiss) is not the right word for this… 🤔

Reply:
I have asked ChatGPT to suggest some other names, here are a few that I consider usable: `visibilityOnLimit`, `visibilityOnLimitReached`, `maxUploadedLimitBehavior`, `uploadLimitBehavior`, and `maxUploadedFilesAction`.

Reply:
The default state probably shouldn't be 'hidden'. On the contrary, there should be one more value, perhaps 'none', or better `undefined'. Because the user can not specify `queueLimitBehavior` at all and nothing will happen but throw errors. But if I set 'hide' or 'disable' it will either be hidden or disabled

Reply:
What do you think about the default value for this should be `disabled`? Or it is correct that the actual state (meaning without any visual change after the limit is reached) is ok and then there should be the default value of `undefined`?

Reply:
I set there as default value 'none' for internal states, then it's like `undefined`